lms/static/sass/shared/_header.scss

```diff
@@ -145,6 +145,30 @@ header.global {
           }
         }
       }
+
+      .wrapper-more-actions {
+
+        // Some overrides and customization for the button
+        // in this particular use case
+        .button-more {
+          @include margin-left(-1px); // Don't want a double border on the side with the profile name
+          @include border-top-left-radius(0);
+          @include border-bottom-left-radius(0);
+          border-color: $gray-l3;
+          height: 34px;
+          line-height: 34px;
+          width: 34px;
+
+          .icon {
+            position: relative;
+            top: -3px;
```

Comment:
The way this lil icon is made requires it to be shifted up slightly to center it vertically.
![screen shot 2015-05-12 at 11 39 54 am](https://cloud.githubusercontent.com/assets/2112024/7591409/bc076840-f89b-11e4-9ae4-b143e3a67a73.png)